core/change.go

```diff
@@ -17,6 +18,14 @@ type ChangesConfigSorter struct {
 	config  Config
 }
 
+type ErrInvalidKind struct {
```

Comment:
I don't think we need a struct here. I would just do `var errInvalidKind = errors.New(...)`
To add the extra info of what kind was attempted use `fmt.Errorf("%w: ...")`

Reply:
Done.